vendor/github.com/openshift/api/operator/v1/0000_70_console-operator.crd.yaml

```diff
@@ -0,0 +1,197 @@
+apiVersion: apiextensions.k8s.io/v1beta1
```

Comment:
this new file is intentional?

Reply:
This was added in the mean time to openshift/api so it will be here as well.
/hold cancel

Reply:
We don't copy anything from operator/v1, and only one specific file from operator/v1alpha1 https://github.com/openshift/cluster-config-operator/blob/master/Dockerfile#L14